.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
+- repo: git@github.com:snowflakedb/casec_precommit.git # SSH
+# - repo: https://github.com/snowflakedb/casec_precommit.git # HTTPS
+  rev: HEAD
+  hooks:
```

Comment:
Is this suppose to run as GH actions?

Reply:
it runs as a local pre-commit hook. see https://pre-commit.com/ for more info.

Reply:
https://travis-ci.org/github/snowflakedb/snowflake-ingest-java/builds/755417043
Does it only run locally?

Reply:
yes, it only runs on a developer's laptop (mac or devvm). It will run as a prerequisite to `git commit` command and it will abort commit if the script finds a secret in the commit diff